pages/common/rustup.md

```diff
@@ -27,3 +27,7 @@
 - Run cargo build with a certain toolchain:
 
 `rustup run {{toolchain_name}} cargo build`
+
+- Open the local rust documentation in your browser:
```

Comment:
```suggestion
- Open the local rust documentation the system default browser:
```

Reply:
There is a typo in your suggested change. You forgot the word 'in'.